server/meca/file-generators/manifest.js

```diff
@@ -1,7 +1,40 @@
 const fs = require('fs-extra')
 
-async function generateManifest() {
-  return fs.readFile(`${__dirname}/manifest.xml`, 'utf8')
+function supplementaryXml(files) {
+  const keyList = ['id', 'mimeType', 'filename']
+  const fileList = files.filter(
+    fileObject => fileObject.type === 'SUPPORTING_FILE',
+  )
+
+  let supplementaryFileXml = ''
+  const template = fs.readFileSync(
```

Comment:
Given that this is `sync`, did we see any delays when submitting?

Reply:
The actual export function is async... 
```

async function mecaExport(manuscript, content, clientIp) {
  logger.info(`Starting MECA export`, { manuscriptId: manuscript.id })
  const archive = await generate(manuscript, content, clientIp)
  await upload(archive, manuscript.id)
}
```